modules/javafx.media/src/main/native/jfxmedia/platform/osx/avf/AVFMediaPlayer.mm

```diff
@@ -85,6 +85,9 @@ static void append_log(NSMutableString *s, NSString *fmt, ...) {
 #define TRACK_LOG(...) {}
 #endif
 
+// Max number of bytes we will provide per request
+#define MAX_READ_SIZE 1048576
```

Comment:
Optional suggestion: define this as `(1024 * 1024)` ? It's OK to leave it as is if you prefer. Maybe add a comment in that case?

Reply:
Fixed.